report-service/src/test/kotlin/ru/morkovka/report/controller/ReleaseServiceImplTest.kt

```diff
@@ -0,0 +1,49 @@
+package ru.morkovka.report.controller
```

Comment:
Этот пакет относится к тестам контроллеров. Если хотим создать тест для сервиса, его стоит положить в соответствующий пакет
```suggestion
package ru.morkovka.report.service.impl
```